core/src/test/java/com/linecorp/armeria/client/endpoint/PropertiesEndpointGroupTest.java

```diff
@@ -42,7 +43,12 @@
 
     @BeforeClass
     public static void before() {
-        Awaitility.setDefaultTimeout(1, TimeUnit.MINUTES);
+        Awaitility.setDefaultTimeout(1, TimeUnit.SECONDS);
```

Comment:
Is this intended? It seems to me that you don't need to adjust the timeout at all if you are decreasing the timeout.

Reply:
whoops! fixed